spur_bringup/package.xml

```diff
@@ -16,11 +16,11 @@
   <buildtool_depend>catkin</buildtool_depend>
   <build_depend>rostest</build_depend>
   <run_depend>dynamixel_controllers</run_depend>
+  <run_depend>hokuyo_node</run_depend>
```

Comment:
from https://github.com/tork-a/spur/pull/49/files#r34216027
we do not have to add one